### DISCUSSION TITLE
868 MHz to 433 MHz mod

### DISCUSSION THREAD

**Discussion by mpiotrowicz7** [dl9rdz/rdz_ttgo_sonde#257] [TTGO hardware]
Could someone please elaborate on:

> The 868/915 MHz version contains the SX1276 receiver chip. The chip supports both 433 MHZ and 868/915 MHz, but using different input pins. The board connects the antenna to the 868/915 MHZ input pin, the 433 MHz pin is unconnected. Theoretically the 868/915 MHZ version can be modified to work as 403 MHz receiver by directly connecting the antenna to the right pin of the chip.

What are the modifications that need to be done on a TTGO v1.0 board?

**Comment by dl9rdz** [comment]
For any board in the 868 MHZ version:

- You have to locate the part with the TTGO LoRa sticker on it (below the sticker it is labelled HPD13A for the 868MHZ version - remove the sticker, otherwise it will melt in the next step).
- Desolder the metal case of the HPD13A
- Below the metal cover you'll find something like a rfm92w board
- This board contains an SX1276 chip (QFN28 casing)
- Using your extremely sophisticated soldering skills, connect something to the Low-RF input pin so you can feed your antenna signal to that pin. 

Not easy. Not tested (by myself, but I heard of someone doing it successfully before). Not recommended (simply by a new board in the right version. It is easier)

**Reply by mpiotrowicz7** [reply]
So the antenna signal should be connected directly to RFI_LF pin. Do I understand correctly. Soldering does not bother me, I was dealing with much worse cases ;)

**Comment by mpiotrowicz7** [comment]
I have also another thought in this topic. I understand I have the incorrect version of TTGO, with regards to the LoRa frequency. However, when I upload a binary, the OLED stays dark and the device does not seem to work properly (or at all). Would it be caused by an incorrect frequency of LoRa?
